distributed-databases/trino/resources.tf

```diff
@@ -0,0 +1,156 @@
+#---------------------------------------------------------------
```

Comment:
Rename this file from `resources.tf` to `trino.tf`. 

Change the name of the resources.tf file to trino.tf.

Then, transfer all Trino-related resources, including the deployment of the Trino add-on module, into this newly renamed `trino.tf` file.

Reply:
addressed